pom.xml

```diff
@@ -909,6 +958,18 @@
             <groupId>commons-logging</groupId>
             <artifactId>commons-logging</artifactId>
           </exclusion>
+          <exclusion>
```

Comment:
It would be good to at least make sure YARN submission works outside unit tests. @mccheah can you do that? Otherwise I can try to find some time to build and test this.